lib/device.c

```diff
@@ -387,6 +387,34 @@ int sync_save_tracks(const struct sync_device *d)
 
 #ifndef SYNC_PLAYER
 
+static int send_set_key_cmd(const struct sync_device *d, const struct sync_track *t, int row, float val, enum key_type type)
+{
+	unsigned char cmd = SET_KEY, _type = type;
+	uint32_t _track, _row;
```

Comment:
We don't start symbol names with underscores. These symbol names are reserved for the implementation (compiler and runtime libraries).

Reply:
Renamed to n prefixes (for network) instead of _.